extensions/interactions/ItemSelectionInput/directives/item-selection-input-validation.service.ts

```diff
@@ -134,9 +134,13 @@ export class ItemSelectionInputValidationService {
 
     var areAllChoicesCovered = false;
     if (maxAllowedCount === 1) {
-      var answerChoiceToIndex = {};
+      var answerChoiceToIndex: Record<string, number> = {};
       seenChoices.forEach((seenChoice, choiceIndex) => {
-        answerChoiceToIndex[seenChoice.contentId] = choiceIndex;
+        const contentId = seenChoice.contentId;
+        if (contentId === null) {
+          throw new Error('ContentId does not exist');
+        }
+        answerChoiceToIndex[contentId] = choiceIndex;
```

Comment:
seems like you haven't added the corresponding test for this error in the spec file

Reply:
done